src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor.cs

```diff
@@ -4,13 +4,14 @@
 using Microsoft.AspNetCore.Components;
 using Microsoft.JSInterop;
 using Bit.Platform.WebSite.Web.Shared;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Bit.Platform.WebSite.Web.Components;
 
 public partial class GettingStarted
 {
-    [Inject] public NavigationManager NavigationManager { get; set; }
-    [Inject] public IJSRuntime JSRuntime { get; set; }
+    [AutoInject] public NavigationManager NavigationManager { get; set; }
+    [AutoInject] public IJSRuntime JSRuntime { get; set; }
```

Comment:
please define these injected properties as `private` and change the property to a field like this:

```cs
[AutoInject] private IJSRuntime _jsRuntime = default!
```

(please apply these changes everywhere in this PR)